modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -5,41 +5,53 @@
-. The *Parameters* for a Task are prepopulated based on the specifications of the Task. If required, use the *Add Parameters* link to add additional parameters.
+... Add an input resource. In the *Name* field, enter `Source`, and from the *Resource Type* drop-down list, select *Git*.
+... Add an output resource. In the *Name* field, enter `Img`, and from the *Resource Type* drop-down list, select *Image*.
+.. Optional: The *Parameters* for a task are prepopulated based on the specifications of the task. If required, use the *Add Parameters* link to add additional parameters.
 +
-. A *Missing Resources* warning is displayed on a Task if the resources for the Task are not specified. Click the *s2i-nodejs* Task to see the side panel with details for the Task.
+.. A *Missing Resources* warning is displayed on a task if the resources for the task are not specified. Click the *s2i-nodejs* task to see the side panel with details for the task.
 +
-.Tasks details in Pipelines Builder
+.tasks details in Pipelines Builder
 image::op-pipeline-builder-task-details.png[]
 +
-. In the Task side panel, specify the resources and parameters for it:
-
-.. In the *Input Resources* -> *Source* section, the *Select Resources* drop-down list displays the resources that you added to the Pipeline. For this example, select *Source*.
-.. In the *Output Resources* -> *Image* section, click the *Select Resources* list, and select *Img*.
-.. If required, in the *Parameters* section, add more parameters to the default ones, by using the `$(params.<param-name>)` syntax.
-. Similarly, add an input resource for the *openshift-client* Task.
-. Click *Create* to create the Pipeline. You are redirected to the *Pipeline Details* page that displays the details of the created Pipeline. You can now use the *Action* button to start the Pipeline.
+.. In the task side panel, specify the resources and parameters for the *s2i-nodejs* task:
 
-Optionally, you can also use the *Edit YAML* link, on the upper right of the *Pipeline Builder* page, to directly modify a Pipeline YAML file in the console. You can also use the operator-installed, reusable snippets and samples to create detailed Pipelines.
+... In the *Input Resources* -> *Source* section, the *Select Resources* drop-down list displays the resources that you added to the pipeline. For this example, select *Source*.
+... In the *Output Resources* -> *Image* section, click the *Select Resources* list, and select *Img*.
+... If required, in the *Parameters* section, add more parameters to the default ones, by using the `$(params.<param-name>)` syntax.
+... Similarly, add an input resource for the *openshift-client* task.
+. Click *Create* to create the pipeline. You are redirected to the *Pipeline Details* page that displays the details of the created pipeline. You can now use the *Action* button to start the pipeline.
```

Comment:
ISG:

> If possible, avoid describing the result of each step:
If the context is clear, omit any locational information.
If the context is not clear, in the next step, orient the user to the location where that step is performed.

Reply:
Will rephrase the sentence as per ISG suggestion.